include/uv/win.h

```diff
@@ -498,7 +498,7 @@ typedef struct {
       uv_buf_t read_line_buffer;                                              \
       HANDLE read_raw_wait;                                                   \
       /* Fields used for translating win keystrokes into vt100 characters */  \
-      char last_key[8];                                                       \
+      char last_key[32];                                                      \
```

Comment:
Since this breaks ABI it can't land on the v1.x branch, only on master. You could perhaps turn this into a pointer and malloc the memory.

Reply:
> You could perhaps turn this into a pointer and malloc the memory.

Are you saying to replace it with a pointer to maintain ABI compatibility(In the case of 32bits, obviously replace it with a pointer and padding of 4 bytes)?

Reply:
That's right.